lerna.json

```diff
@@ -9,6 +8,9 @@
   ],
   "command": {
     "publish": {},
-    "bootstrap": {}
+    "bootstrap": {},
+    "init": {
+      "exact": true
```

Comment:
The lerna options syntax is not the most transparent. Do we still need `"exact"` on line 5?